src/api/interactivity.js

```diff
@@ -62,9 +62,9 @@ export default class Interactivity {
     }
 
     _getFeaturesAtPosition(lngLat) {
-        const wm = projectToWebMercator({ lng: lngLat[0], lat: lngLat[1] });
+        const wm = projectToWebMercator({ lng: lngLat.lng, lat: lngLat.lat });
         const nwmc = wToR(wm.x, wm.y, { scale: WM_R, center: { x: 0, y: 0 } });
-        return this._layerList.map(layer => layer.getFeaturesAtPosition(nwmc)).reduce(Array.concat);
+        return [].concat(...this._layerList.map(layer => layer.getFeaturesAtPosition(nwmc)));
```

Comment:
utils.flatten ?

Reply:
`[].concat(...` this is as long as `utils.flatten`, and I prefer a native form over a custom function call